sim/midas/src/main/cc/simif_emul.cc

```diff
@@ -142,13 +142,15 @@ void simif_emul_t::wait_read(std::unique_ptr<mmio_t>& mmio, void *data) {
 
 void simif_emul_t::write(size_t addr, data_t data) {
   size_t strb = (1 << CTRL_STRB_BITS) - 1;
-  master->write_req(addr << CHANNEL_SIZE, CHANNEL_SIZE, 0, &data, &strb);
+  // LS by 2 because the driver presents word addresses, but AXI4-lite expects
+  // byte addresses
+  master->write_req(addr << 2, 2, 0, &data, &strb);
```

Comment:
Is there a reason to drop the use of CHANNEL_SIZE as a stand-in for the literal instead of just moving the definition?

Reply:
It used to refer to the maximum AXI4 size of the control interface. I should probably just add it back with a better name since there exists no global notion of a "Channel's width" now.